docs/topic-guides/users.txt

```diff
@@ -3,39 +3,44 @@
 Users
 =====
 
-Determined is designed for teams of machine learning developers. By default,
-developers can access Determined without needing to authenticate; moreover,
-the objects (e.g., experiments and notebooks) created by one developer
-will be visible to all users of the system.
+Determined is designed for teams of machine learning developers. Objects, such as experiments
+and notebooks, created by a developer will be visible to all users of the system.
 
-While this is convenient for small teams, larger teams may prefer to
-create a Determined *user* account for each developer. After authenticating as
-a given user, the Determined objects created in that session will be
-associated with that user account. By default, the WebUI and CLI will
-only display objects created by the current user.
+.. warning::
 
-Initially, a Determined installation has two users: ``admin`` and ``determined``.
-The ``determined`` user is the default account that is used for WebUI and CLI
-operations. If individual user accounts are not required, no additional
-configuration is needed. In this case, all assets created within Determined
-(experiments, commands, etc.) will be attributed to ``determined``.
+   The ``admin`` and ``determined`` users initially have blank passwords. Use the ``det user
+   change-password`` to set a password for the ``admin``.
 
-The ``admin`` user can be used to create users, change other users'
-passwords, and activate/deactivate users.
+   .. code::
 
-.. note::
+      det -u admin user change-password
 
-   The ``admin`` user initially has a blank password. The ``user
-   change-password`` CLI command can be used by the admin to update this
-   password:
+   We also suggest deactivating the user the user ``determined``:
```

Comment:
blocking: This is a little confusing -- if people are supposed to deactivate the user, what's the alternative for logging in?